fetch.bs

```diff
@@ -2133,6 +2133,19 @@ Unless stated otherwise, it is false.
 
 <p class=note>This flag is for exclusive use by HTML's render-blocking mechanism. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated boolean <dfn export for=request>https-upgrade</dfn>.
+Unless stated otherwise, it is false.
+
+<p class=note>This flag is for exclusive use by https-upgrades algorithm.
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=https-upgrade-fallback-url>https-upgrade-fallback-url</dfn>. It is a copy
+of the <a for=/>request</a>'s original <a for=/>URL</a> before <a for=/>request</a> is
```

Comment:
`for=request>URL`, no?

Reply:
I thought the idea was that with synthetic redirects the initial URL would not be modified, but maybe the idea of synthetic redirects was abandoned?

Reply:
for=request>URL Done 

Not sure about synthetic redirects. In fact I can't find any docs about synthetic redirects so I'd appreciate pointers.

Reply:
I recommend reaching out to @davidben regarding this. See also https://github.com/whatwg/fetch/issues/1426.